### DISCUSSION TITLE
calibration rejected Unknown code: 17

### DISCUSSION THREAD

**Discussion by Stormmm41** [NightscoutFoundation/xDrip#3409] [Bug reports]
Hi, I'm trying to calibrate my sensor (dexcom G6). And this error has appeared many times already; this has never happened before. The problem appeared after replacing the sensor.

**Comment by Navid200** [comment]
Please post a screenshot of this status page: https://navid200.github.io/xDrip/docs/StatusG5G6.html

In absence of that, what I have related is this: https://navid200.github.io/xDrip/docs/Calibrate-G6.html
